cms/static/js/certificates/models/certificate.js

```diff
@@ -20,9 +20,25 @@ function (_, str, Backbone, BackboneRelational, BackboneAssociations, gettext, C
         defaults: {
             name: 'Name of the certificate',
             description: 'Description of the certificate',
-            course_title: 'Title of the course',
+
+            /**
+             * It represents the course title.
+             */
+            course_title: '',
+
+            /**
+             * It represents the path to organization logo image.
+             */
             org_logo_path: '',
+
+            /**
+             * It represents the default version of the certificate.
```

Comment:
@catong -- this is an internal versioning counter used to ensure the requests being made by the UI to the server are of the correct format/schema